Dockerfile

```diff
@@ -3,14 +3,21 @@ FROM docker.io/centos:centos7
 RUN yum install -y python-requests && \
     curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b stein current-tripleo && \
     yum update -y && \
-    yum install -y openstack-ironic-api openstack-ironic-conductor crudini \
-        iproute iptables dnsmasq httpd qemu-img-ev iscsi-initiator-utils parted gdisk ipxe-bootimgs psmisc sysvinit-tools \
-        mariadb-server python-PyMySQL python2-chardet && \
+    yum install -y crudini iproute iptables dnsmasq httpd qemu-img-ev \
+        iscsi-initiator-utils parted gdisk ipxe-bootimgs psmisc \
+        sysvinit-tools mariadb-server python-PyMySQL python2-chardet \
+        python-pip git gcc python-devel && \
     yum clean all
 
+COPY ./actual-reqs.txt /var/tmp/actual-reqs.txt
+COPY ./installer.sh /bin/installer.sh
+
+RUN chmod +x /bin/installer.sh && /bin/installer.sh
+
 RUN mkdir /tftpboot && \
     cp /usr/share/ipxe/undionly.kpxe /usr/share/ipxe/ipxe.efi /tftpboot/
 
+RUN mkdir -p /etc/ironic
 COPY ./ironic.conf /tmp/ironic.conf
 RUN crudini --merge /etc/ironic/ironic.conf < /tmp/ironic.conf && \
     rm /tmp/ironic.conf
```

Comment:
remove gcc and other build dependencies at some point

Reply:
yep, true

Reply:
done